docs/events.yaml

```diff
@@ -408,20 +408,6 @@ x-events:
   properties:
     infra_env_id: UUID
 
-- name: discovery_ignition_config_applied
-  message: "Custom discovery ignition config was applied to the cluster"
```

Comment:
Do we have an analogous event for infra-env?
I think this would be good to see in events.

Reply:
Good question, now that it's part of infra env user will probably get an even when updating it no? 
@avishayt maybe it's something that we need to consider, do we want to publish another event for specific updates? 
Or is the message in update event should include additional information about things that have been updated?

Reply:
I think that for the regular update API it would be messy to list all the changes.  If there is something out of the ordinary (like customizations) then it's important for support and we should have events, but changing network or hostname doesn't need it.

Reply:
Currently, `UpdateInfraEnv()` (which replaced `UpdateDiscoveryIgnition`) produce `ignition_config_image_generated` event for success: 
https://github.com/openshift/assisted-service/blob/483d52644b50f652f6c96e6f614734e24f3353ad/docs/events.yaml#L584-L590

and `upload_image_failed` in a case of failure:
https://github.com/openshift/assisted-service/blob/483d52644b50f652f6c96e6f614734e24f3353ad/docs/events.yaml#L577-L582

The `ignition_config_image_generated` event is different from `discovery_ignition_config_applied` and `ignition_updated_therefore_image_deleted` has no replacement here, probably because of the changes we had to work with the image-service.

Whould we add more events? and if yes, should it be a part of this PR?

Reply:
I don't think we're actually ever sending either of those events because of the early return for the image service here: https://github.com/openshift/assisted-service/blob/01aef1f8361609863f4e97cf7b77d0b18447150c/internal/bminventory/inventory.go#L1166-L1172

There's a LOT of dead code in there that I want to get to cleaning up after the main v1 removal is done.